query-builder/src/main/java/com/datastax/oss/driver/api/querybuilder/insert/InsertInto.java

```diff
@@ -40,15 +38,6 @@
   /**
    * Makes this statement an INSERT JSON with a custom type mapping. The provided {@code Object
    * value} will be mapped to a JSON string.
-   *
-   * <p>This is an alternative to {@link #json(String)} for custom type mappings. The provided
-   * registry should contain a codec that can format the value. Typically, this will be your
-   * session's registry, which is accessible via {@code session.getContext().getCodecRegistry()}.
-   *
-   * @throws CodecNotFoundException if {@code codecRegistry} does not contain any codec that can
-   *     handle {@code value}.
-   * @see DriverContext#getCodecRegistry()
-   * @see DefaultInsert#json(Object, CodecRegistry)
```

Comment:
Why remove so much? I think you could remove only the offending line: `@see DefaultInsert#json(Object, CodecRegistry)`

Reply:
Ok, I thought that all `@see` and `code` are problematic. Reverted